internal/namespaces/instance/v1/custom_image.go

```diff
@@ -314,6 +315,125 @@ func imageDeleteBuilder(c *core.Command) *core.Command {
 	return c
 }
 
+//
```

Comment:
Why is it not generated?

Reply:
Because the [UpdateImage](https://github.com/scaleway/scaleway-sdk-go/blob/master/api/instance/v1/image_utils.go#L81) method has been hand coded in the SDK

Reply:
Could we check with the instance team if the update image could be generated now? Maybe the reasons of why it was hardcoded are no long true and it could be generated just like any other function

Reply:
@kindermoumoute